src/main/com/diana/restaurant/queue/QueueManager.java

```diff
@@ -0,0 +1,33 @@
+package com.diana.restaurant.queue;
+
+import com.diana.restaurant.entities.Order;
+import com.diana.restaurant.enums.Queues;
+import com.diana.restaurant.exceptions.queueManager.QueueManagerKeyNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentLinkedQueue;
+
+public class QueueManager {
+  private static QueueManager instance;
+  private final Map<String, Object> instanceMap = new HashMap<>();
```

Comment:
Why are you using `Object` as the map value?

Reply:
solved